cypress/integration/dimensions/calculations.cy.js

```diff
@@ -0,0 +1,102 @@
+import { DIMENSION_ID_DATA } from '@dhis2/analytics'
+import {
+    clickNewCalculationButton,
+    expectCalculationsModalTitleToContain,
+    expectCalculationsModalToBeVisible,
+} from '../../elements/calculationsModal.js'
+import { expectDataDimensionModalToBeVisible } from '../../elements/dimensionModal/index.js'
+import { openDimension } from '../../elements/dimensionsPanel.js'
+import { goToStartPage } from '../../elements/startScreen.js'
+
+const PAGE_SIZE = 50
+const DATA_ELEMENTS_URL = '**/dataElements?*'
+
+describe('Data dimension', () => {
+    describe('initial state', () => {
+        it('navigates to the start page', () => {
```

Comment:
The Cypress tests are still WIP, there's a whole list of what needs to be done further down in the file. I haven't settled for if I should use the style from DV (1 `it` per step) or LL (1 `it` per test`).
However I assume there's going to be quite a lot of repetition though (like other tests in DV), so I hope to extract some reusable parts to avoid copy/pasting code (which is in favour of the DV style).

Reply:
The tests are updated now as suggested, went with the "LL style", i.e. each `it` starts from scratch and can be run individually